client/my-sites/stats/activity-log-banner/success-banner.jsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import ActivityLogBanner from './activity-log-banner';
+import Button from 'components/button';
+
+function SuccessBanner( {
+	translate,
+	moment,
+} ) {
+	// FIXME: real dates
+	const date = 1496400468285;
+	// FIXME: real dismiss
+	const handleDismiss = () => {};
+
+	return (
+		<ActivityLogBanner
+			isDismissable
+			onDismissClick={ handleDismiss }
+			status="success"
+			title={ translate( 'Your site has been successfully restored' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **22** times:
`translate( 'has been successfully restored!' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).